tomcat-main/src/main/resources/defaults/base-collectionobject.xml

```diff
@@ -393,6 +397,10 @@
 			</repeat>
 			<field id="assocEventName" />
 			<field id="assocEventNameType" />
+
+			<field id="assocControlledEventName" />
+			<field id="assocControlledEventNameType" />
```

Comment:
I know you were following the example of `assocEventName`/`assocEventNameType`, but I don't know why they weren't just `assocEvent`/`assocEventType`. It's `assocPerson`, `assocPlace`, etc., not `assocPersonName`, `assocPlaceName`, etc. I tend to think we should go without the "Name" for these new ones. So `assocControlledEvent`/`assocControlledEventType`.

Edit: I wonder if "EventName" was used in the old fields, because it was expected that there would be controlled events in the future? Maybe "Name" meant just a string, vs. a reference to an authority item. In which case, maybe the intent was that these authority-controlled fields would just be named `assocEvent` and `assocEventType`. I'm ok with that too.

Reply:
The `assocEvent` and `assocEventType` are nice, I don't think the explicit `controlled` adds too much. Also looks like I forgot to add the `autocomplete` onto the field so I'll get that fixed.